src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -525,23 +527,42 @@
     </script>
 
     <!-- TOP BAR, FIXED TO THE TOP OF THE SCREEN -->
-    <div class="fixed">
-        <nav class="top-bar">
-            <ul>
-                <li class="name">
-                    <h1>@OptionalLinkText(@Model.TitleMain)</h1>
-                    <h2>@OptionalLinkText(@Model.TitleDetail)</h2>
-                </li>
-                <!-- USED FOR SMALLER SCREENS TO CONDENSE NAVIGATION -->
-                <li class="toggle-topbar"><a href="javascript:;"></a></li>
+    <header>
+        <div class="dropdown header-dropdown nav-main-dropdown">
+            <button class="button dropdown-toggle nav-main-button" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+                <div class="nav-main">
+                    <div class="nav-main-title">
+                        <h1>@OptionalLinkText(@Model.TitleMain)</h1>
+                    </div>
+                    <i class="dropdown-icon icon-menu"></i>
+                </div>
+            </button>
+            <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
+                @RenderLinks(Model.HeaderLinks)
             </ul>
-            <section>
-                <ul class="right">
-                    @RenderLinks(Model.HeaderLinks)
-                </ul>
-            </section>
-        </nav>
-    </div>
+        </div>
+        <div class="dropdown header-dropdown nav-locations">
+            <button class="button dropdown-toggle nav-main-button" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
```

Comment:
`dropdownMenu1` from the prototype is used as an id on a few different elements in this PR.  Consider changing the id/labeledby sets to more meaningful and unique names.